data/styles/default/dark/symbols/monorail-m.svg

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg width="100%" height="100%" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" xmlns:serif="http://www.serif.com/" style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:2;">
```

Comment:
please add `width="18" height="18"` (helps to preview in real size)
also maybe cleanup unnecessary tags a bit, see e.g. `bus-m.svg`
:pray:

Reply:
fixed width and height values, removed unnecessary attributes and and DOCTYPE tag